### PR TITLE
Added KLDLoad and updated modprobeBSD

### PR DESCRIPTION
**Pull request by r-dailey**
Added a KLDLoad tool to the kernel config tool set. Additionally added a load function to the modprobeBSD tool for managing modules during BSD testing.

### CODE REVIEW

File: lisa/tools/kernel_config.py [hunk 1]
```diff
@@ -133,3 +133,30 @@ def module_exists(
             return True
 
         return False
+
+
+class KLDLoad(Tool):
```
Comment (squirrelsc) [inline lisa/tools/kernel_config.py]: If the class is used in modprobe only, it should be in modeprobe.

Comment (r-dailey) [reply lisa/tools/kernel_config.py]: I was following the format of KLDStat as that is a similar tool. KLDStat is only used in modprobe from the references I can see in VScode. Should I move both KLDLoad and KLDStat into modprobe?

Comment (squirrelsc) [reply lisa/tools/kernel_config.py]: Yes, please move them both.

Comment (r-dailey) [reply lisa/tools/kernel_config.py]: I missed one use of kldstat in kernelconfigBSD so I left it there. KLDLoad is now in modprobe.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**